server/controllers/notes.js

```diff
@@ -0,0 +1,20 @@
+const Notes = require('../models/notes')
+
+const notesController = {
+    create: (req, res) => {
+        // code here
+    },
+
+    update: (req, res) => {
+
+    },
+
+    delete: (req, res) => {
+
+    },
+
+    insert: (req, res) => {
```

Comment:
create is similar as insert, lets put all methods listed below in routes

Reply:
ok